my-react-app/src/components/inputComponents/inputCheckbox/InputCheckbox.tsx

```diff
@@ -0,0 +1,40 @@
+import { InputAdditionalServiceInterface } from 'models/models';
+import React from 'react';
+
+export function InputCheckbox({ register }: InputAdditionalServiceInterface) {
+  return (
+    <div className="form__additional form-block">
+      Additional options:
+      <label className="form__additional--label form-label" htmlFor="form__package">
+        <input
+          className="form__additional--input form-input"
+          type="checkbox"
+          id="form__package"
+          value=" package "
+          {...register}
+        />
+        Package
+      </label>
+      <label className="form__additional form-label" htmlFor="form__delivery">
```

Comment:
this label already assosiated with nested input
```suggestion
      <label className="form__additional form-label">
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label